zerver/lib/markdown/preprocessor_priorities.py

```diff
@@ -1,6 +1,7 @@
 # Note that in the Markdown preprocessor registry, the highest
 # numeric value is considered the highest priority, so the dict
 # below is ordered from highest-to-lowest priority.
+# Built-in preprocessor priorities are commented out.
```

Comment:
Change this comment to `Priorities for the built-in preprocessors are commented out.`

Reply:
Alright, that should be it. Thanks for the help!